js/search.js

```diff
@@ -1,80 +1,7 @@
-
-function get_score(key, result) {
```

Comment:
Moved functions out of global scope to avoid name collisions. They are only used inside `function search(searchfield_id, searchresult_id)`.